app/legacy_lib/insert_card.rb

```diff
@@ -92,11 +90,7 @@ def self.with_stripe(card_data, stripe_account_id=nil, event_id=nil, current_use
     source_token = nil
     begin
       Card.transaction {
-
-        if (holder_type == :nonprofit)
-          # @type [Nonprofit] holder
-          card = holder.create_active_card(card_data)
-        elsif (holder_type == :supporter)
+        if (holder_type == :supporter)
```

Comment:
Please remove if conditional because there's no other type of holder than `:supporter` going forward